lang/lua-src/README.md

```diff
@@ -0,0 +1,51 @@
+# Usage
+
+Run the `./script.sh` script.
+This will download, extract and build `lua` in the `app-lua/` directory.
+
+## Linux
+
+When inside `app-lua`
+
+```console
+./lua helloworld.lua
+Hello World
+```
+
+With no arguments, it will open the Lua console.
+
+```console
+> 1
+1
+> 1+1
+2
+```
+
+## Unikraft
+
+```console
+git clone https://github.com/unikraft/run-app-elfloader app-elfloader
+cd app-elfloader
+./run.sh -r {PATH_TO_app-lua} /lua [file.lua]
+```
+
+`file.lua` needs to be placed in the `app-lua` directory.
```

Comment:
Could you provide an example with the `1  + 1` console example for Unikraft,  to have something similar with the Linux section above